spec/models/role_spec.rb

```diff
@@ -17,35 +17,35 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #++
 
-require File.expand_path(File.dirname(__FILE__) + "/../spec_helper")
+require File.expand_path(File.dirname(__FILE__) + '/../spec_helper')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.